.env.development

```
@@ -1,7 +1,7 @@
 SEED_ADMIN_EMAIL=admin@example.com
-SEED_ADMIN_INITIAL_PASSWORD=password
+SEED_ADMIN_INITIAL_PASSWORD=F8osN3hiRjJ0dstGJUDR
```

I prefer something easier to type for the cases where you want to test on a device without access to copy paste from the file
```suggestion
SEED_ADMIN_INITIAL_PASSWORD=`liga guide prep mare`
```

Although, I think the decision at our last Muffi meeting was to not change the password...

---

Let keep "password"

---

due to pwned password protection we cant use "password" anymore, so i'm ok with using random word group instead

---

I've updated it to be word string, however if we add strong password validation it wont pass due to lack of numbers/uppercase letters